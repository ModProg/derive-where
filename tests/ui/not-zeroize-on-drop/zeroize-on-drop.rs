extern crate zeroize_ as zeroize;

use std::marker::PhantomData;

use derive_where::derive_where;

#[derive_where(ZeroizeOnDrop(no_drop))]
struct InvalidOption<T>(PhantomData<T>);

fn main() {}
