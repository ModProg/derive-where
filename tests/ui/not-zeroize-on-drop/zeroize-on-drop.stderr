error: requires crate feature `zeroize-on-drop`
 --> tests/ui/not-zeroize-on-drop/zeroize-on-drop.rs:7:30
  |
7 | #[derive_where(ZeroizeOnDrop(no_drop))]
  |                              ^^^^^^^
